src/include/duckdb/common/types/string_type.hpp

```diff
@@ -65,7 +65,12 @@ struct string_t {
 	}
 
 	bool IsInlined() const {
+#ifndef DUCKDB_DEBUG_NO_INLINE
```

Comment:
Is this change necessary as part of this PR? I believe there is a reason zero-length strings are inlined even with the debug setting.